libs/components/code-examples/src/lib/modules/datetime/datepicker/basic/example.component.ts

```diff
@@ -49,16 +49,19 @@ function validateDate(
 })
 export class DatetimeDatepickerBasicExampleComponent {
   protected formGroup: FormGroup<DemoForm>;
-  protected startDate: FormControl<Date | null>;
+  protected startDate: FormControl<Date | null | string>;
   protected hintText = 'Must be before your 1 year anniversary.';
 
   public helpPopoverContent =
     'If you need help with registration, choose a date at least 8 business days after you arrive. The process takes up to 7 business days from the start date.';
 
   constructor() {
-    this.startDate = new FormControl<Date | null>(new Date('10/12/2001'), {
-      validators: [Validators.required, validateDate],
-    });
+    this.startDate = new FormControl<Date | null | string>(
```

Comment:
I'd order these types so that the "unset" value is at the end: `Date | string | null`.